pack/ironheart.json

```diff
@@ -495,14 +495,15 @@
 		"code": "29025",
 		"cost": 3,
 		"deck_limit": 1,
+		"erratum": "Added “Remove ‘Go for Champions!’ from the game →”. (RRG 1.5)",
 		"faction_code": "basic",
 		"name": "\"Go for Champions!\"",
 		"octgn_id": "9d1ed258-ee76-47e7-8f1d-b9dd590fd293",
 		"pack_code": "ironheart",
 		"position": 25,
 		"quantity": 1,
 		"resource_wild": 1,
-		"text": "Max 1 per deck.\nPlay only if your identity has the [[Champion]] trait.\n<b>Hero Action</b>: Each [[champion]] character in play cannot take damage until the end of the round.",
+		"text": "Max 1 per deck.\nPlay only if your identity has the [[Champion]] trait.\n<b>Hero Action</b>: Remove ‘Go for Champions!’ from the game → Each [[champion]] character in play cannot take damage until the end of the round.",
```

Comment:
Single quotes are used inside double quotes to signify a quote within a quote. In this case, I think we want to say `Remove \"Go for Champions!\" from the game` because this card's text value isn't already a quote. It would also better match the title of this card.

I get that this is verbatim from the rules book but given the context of those rules, I think we can't just copy/paste it here. Above in the `erratum`, I think the current value is correct because there's a quote around "Remove ... from the game" so the single quote is correct.